_dev/src/components/product-feed/settings/target-countries/target-countries.vue

```diff
@@ -183,8 +184,13 @@ export default {
     },
     activeCountriesWhereShipppingExist() {
       const arrayOfCountries = [];
+
       this.$store.state.productFeed.settings.deliveryDetails.forEach((carrier) => {
-        arrayOfCountries.push(carrier.country);
+        availableCountries.forEach((country) => {
+          if (country.code === carrier.country) {
+            arrayOfCountries.push(carrier.country);
+          }
+        });
```

Comment:
I think using `this.$store.getters['app/GET_ACTIVE_COUNTRIES_FOR_ACTIVE_CURRENCY'];` as before #1084 would be better.

Reply:
Before your fix, all carriers countries were shown. Now, valid countries for all currencies are shown.

As the objective is to have countries for the current currently AND were there is at least one carrier, it seems interesting to have a test checking this.